src/reverselock.h

```diff
@@ -13,7 +13,7 @@ class reverse_lock
 {
 public:
 
-    explicit reverse_lock(Lock& lock) : lock(lock) {
+    explicit reverse_lock(Lock& _lock) : lock(_lock) {
```

Comment:
Can you also squash it? Otherwise no one can start review to check if the binaries are the same.